code/src/cljs/sixsq/nuvla/ui/apps_application/views.cljs

```diff
@@ -293,8 +276,11 @@
           [ui/TableCell (str/capitalize (@tr [:id]))]
           [ui/TableCell [values/AsLink id :label (general-utils/id->uuid id)]]])
        [ui/TableRow
-        [ui/TableCell (str/capitalize (@tr [:version-number]))]
-        [ui/TableCell version-index " " (up-to-date? version-index @versions-map @is-module-published?)]]
+        [ui/TableCell (str/capitalize (@tr [:app-version]))]
+        [ui/TableCell
+         [module-plugin/LinkToAppView {:path path :version-id version-index} version-index]
```

Comment:
This one should not be a link. Because You are already in the app version. A link to stay at the same place is not useful. Or is it for something else?

Reply:
No, you're right, will remove.